TaylorFramework/Modules/BoundariesKit/Component.swift

```diff
@@ -41,7 +41,7 @@ extension Component : Equatable {}
 func ==(lhs: Component, rhs: Component) -> Bool {
     if lhs.range != rhs.range { return false }
     if lhs.type != rhs.type { return false }
-    if lhs.name !~== lhs.name { return false }
+    if lhs.name != lhs.name { return false }
     if !(Set(lhs.components) == Set(rhs.components)) { return false }
     
     return true
```

Comment:
I doubt it leads to any performance increases as execution in an `and` chain stops when it first encounters a false. The reason why it would be more clear is again, when (scaled) it would be more clear which condition affects the result in what manner. Now we have to look at each return condition